docs/how-to/enable-intel-gpu.rst

```diff
@@ -0,0 +1,97 @@
+.. _enable_intel_gpu:
+
+Enable Intel GPUs
+=============================
+
+This guide describes how to configure DSS to utilise the Intel GPUs on your machine. This can be done by enabling the Intel device plugin on your MicroK8s cluster.
+
+Prerequisites
+-------------
+
+* :ref:`MicroK8s is installed <set_microk8s>`.
+* :ref:`DSS CLI is installed <install_DSS_CLI>`.
+* The Intel GPU is recognised by your machine. This can be confirmed by running `intel_gpu_top` from the `intel-gpu-tools` package. For discrete GPUs on older versions of Ubuntu like 22.04, you may need to perform additional steps like install a HWE kernel.
+
+Install the kubectl snap
```

Comment:
Can we follow the same document structure as the NVIDIA one?
- Install the Intel Device Plugin
    - Have the kubectl subsection there
    - Have the commands for verifying the Intel GPU is visible (commands on `intel_gpu_top` and note about HWE here?)
- Verify the Intel Device Plugin is up
- Verify the DSS detects the GPU

cc @afgambin

Reply:
Yes, pls